core/src/main/java/org/hibernate/ogm/query/impl/OgmQueryTranslator.java

```diff
@@ -80,9 +80,17 @@
 	 */
 	private SelectClause selectClause;
 
+	/**
+	 * When the query is only targeting one type, we register the relevant information here.
+	 */
 	private EntityKeyMetadata singleEntityKeyMetadata;
 
 	/**
+	 * When the query is only targeting one type, we register the relevangt information here.
```

Comment:
s/relevangt/relevant/

Not considering the syntax error could you add some details about what you mean with "relevant"? In particular, I don't think I understand the difference between, singleIndexedTypeName and singleEntityKeyMetadata. (Why the word "indexed"?)